e2e/cypress/integration/team/teams_spec.js

```diff
@@ -94,35 +93,69 @@ describe('Teams Suite', () => {
             // * Verify user is on the list, then select by clicking on it
             cy.wrap($el).get('.users-emails-input__menu').
                 children().should('have.length', 1).
-                eq(0).should('contain', `@${user.username}`).and('contain', `${user.firstName} ${user.lastName}`).
+                eq(0).should('contain', `@${user1.username}`).and('contain', `${user1.firstName} ${user1.lastName}`).
                 click();
         });
 
-        // # Click Invite Members
-        cy.getByText(/Invite Members/).click();
+        // # Click "Invite Members" button, then "Done" button
+        cy.getByText(/Invite Members/).should('be.visible').click();
+        cy.getByText(/Done/).should('be.visible').click();
+
+        // * As sysadmin, verify system message posts in Town Square and Off-Topic
+        cy.getLastPost().wait(TIMEOUTS.TINY).then(($el) => {
```

Comment:
Explicit wait here and other similar cases were added to ensure that the user profile is loaded on the last post.